inc/datahandlers/post.php

```diff
@@ -618,8 +628,6 @@ function verify_prefix()
 		}
 		else
 		{
-			$prefix_cache = build_prefixes($prefix);
```

Comment:
Now that I look again at it, the 2nd removed `build_prefixes();` call is needed as well if the 1st is overwritten by `build_prefixes($prefix)`. The very first `build_prefixes();` (not removed) doesn't seem to be needed though.